apidef/oas/server.go

```diff
@@ -114,17 +118,14 @@ type GatewayTags struct {
 
 func (gt *GatewayTags) Fill(api apidef.APIDefinition) {
 	gt.Enabled = !api.TagsDisabled
-	if api.Tags == nil {
-		api.Tags = []string{}
-	}
 	gt.Tags = api.Tags
+	if gt.Tags == nil {
```

Comment:
We don't need to allocate, removed both cases

Reply:
yes, but schema fails when it is null. I don't know maybe we should change schema

Reply:
it's not null, it's cleared based on len(tags)==0 :+1: